### PR TITLE
Move mongoose dependency to peerDependencies

### PR DESCRIPTION
**Pull request by AndreJoaquim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,19 +26,11 @@
...
-    "cover": "istanbul cover ./node_modules/mocha/bin/_mocha --recursive -- -R spec"
-  },
-  "bugs": {
-    "url": "https://github.com/StreetHub/mongoose-version/issues"
-  },
-  "homepage": "https://github.com/StreetHub/mongoose-version#readme",
-  "directories": {
-    "test": "test"
+  "peerDependencies": {
+    "mongoose": ">= 5.11.10"
```
Comment (rmdp-trouva) [inline package.json:32]: 👍 same on [mongoose upgrade on promenade](https://github.com/Trouva/promenade/pull/2175/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R94)

**Review by rmdp-trouva — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (AndreJoaquim) [inline package.json:32]:
```suggestion
    "mongoose": "^5.11.10"
```

Comment (AndreJoaquim) [reply package.json]: Done! ✅

**Comment by AndreJoaquim** [comment]
Closed in favour of #5

**Pull request closed without merging**